### ISSUE TITLE
Large mesh, some geometry is missing

### ISSUE THREAD

**Issue by dmsurti** [dmsurti/AssimpKit#61]
From #56 tyre 3D model which is a large mesh, some geometry is missing after the fix in #59.

See the screenshot in example app compared with blender:

![screenshot 2017-08-31 15 52 54](https://user-images.githubusercontent.com/4158417/29918878-cd815c8c-8e64-11e7-97d6-9102e431008f.png)

<img width="639" alt="screenshot 2017-08-31 15 52 37" src="https://user-images.githubusercontent.com/4158417/29918959-1108f992-8e65-11e7-8f40-758a5d3569a7.png">

**Comment by beak90** [comment]
I'm seeing this issue with a lot of files. I've attached a zip with a few stl files that have a significant amount of missing geometry. I've tried setting many preprocessing settings, including AssimpKit_Process_FixInfacingNormals, but the issue remains. It does appear that some of the geometry is only showing on one side of the triangles, but other parts of the geometry is missing no matter which side you look at it.
[messed up stls.zip](https://github.com/dmsurti/AssimpKit/files/1363562/messed.up.stls.zip)

Device: iPhone 6s, iOS 11.01

**Comment by tobyduffy** [comment]
I'm also still experiencing this issue, I've attached an .obj that is giving me issues as well as the tire model shown above. Any idea on when v1.3 will be released with this bug fix?

[helicopter.obj.zip](https://github.com/dmsurti/AssimpKit/files/1438225/helicopter.obj.zip)

**Comment by dmsurti** [comment]
@tobyduffy Yes, there is a fix in place, not yet integrated. v1.3.0 is [due for release mid November](https://github.com/dmsurti/AssimpKit/milestone/9).

Thanks for sharing the file.
